docs/migration-7.10-11.md

```diff
@@ -1,115 +1,115 @@
-    - [Wiring Files using the Configuration Framework](#wiring-files-using-the-configuration-framework)
-    - [Adapt Logback Configuration](#adapt-logback-configuration)
-    - [Check remaining staticfiles](#check-remaining-staticfiles)
-    - [Verify and correct dependencies](#verify-and-correct-dependencies)
+   - [Global defined dependencies](#global-defined-dependencies)
+   - [Remove Sites Folder Copy Tasks](#remove-sites-folder-copy-tasks)
+   - [Wiring Files using the Configuration Framework](#wiring-files-using-the-configuration-framework)
+   - [Adapt Logback Configuration](#adapt-logback-configuration)
+   - [Check Remaining Static Files](#check-remaining-static-files)
+   - [Verify and Correct Dependencies](#verify-and-correct-dependencies)
 
 ## Preparation Steps
 
-> **Note:** The following marker are used in the commands below:
-> - `$ICM` is a symbolic marker for the root directory of your ICM 7.10 project
-> - `$ICM_11` is a symbolic marker for the root directory of the ICM 11+ project template
+> **Note:** The following markers are used in the commands below:
+> - `$ICM`: A symbolic marker for the root directory of your ICM 7.10 project
+> - `$ICM_11`: A symbolic marker for the root directory of the ICM 11+ project template
 
-### Prepare ICM11 template
-Retrieve customization template for ICM 11 and follow the prerequisites steps.
+### Prepare ICM 11 Template
+Retrieve the ICM 11 customization template and follow the prerequisite steps.
 
-Use customization template to create initial project structure
+Use the customization template to create the initial project structure.
 
-* checkout customization template
-* follow the documentation to configure the template
-  * use versions for ICM 11 in the first step
-* execute the initialization script
+* Checkout customization template.
+* Follow the documentation to configure the template.
+  * Use versions for ICM 11 in the first step.
+* Execute the initialization script.
 
-As result following files are created
+As a result, the following files are created:
 - `build.gradle.kts` - root build script to configure subprojects (cartridges)
   -- define and apply gradle repositories (allows to download ICM cartridges)
   -- apply version filter to subproject (allows central definition of versions at two subprojects `versions` and `versions_test`)
 - `my_*` directories containing example cartridges for different purposes
 - `ft_production` directory defines the cartridge set of production
 - `ft_test` directory defines the test cartridge set for server tests (mostly test data)
 
-### Verify ICM11 template
+### Verify ICM 11 Template
 To ensure the following migration bases on a working template, verify the following:
-- go to your ICM11+ project
-- check that result of customization template is working
-- set marker
+1. Switch to your ICM 11+ project.
+1. Check that result of customization template is working.
+1. Set marker:
 
 ```
 gradlew compileTestJava
```

Comment:
What exactly does the command do? Only set the marker? It seems in other steps, the bullet points describe what the command does, but here there seem to be previous steps that need to be done.  @fgiesske-intershop